keyboards/protokeeb/rev1/keyboard.json

```diff
@@ -0,0 +1,507 @@
+                    "matrix": [0, 3],
+                    "x": 3,
+                    "y": 0,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "MI_E",
+                    "matrix": [1, 0],
+                    "x": 0,
+                    "y": 1,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "MI_F",
+                    "matrix": [1, 1],
+                    "x": 1,
+                    "y": 1,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "MI_Fs",
+                    "matrix": [1, 2],
+                    "x": 2,
+                    "y": 1,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "MI_G",
+                    "matrix": [1, 3],
+                    "x": 3,
+                    "y": 1,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "MI_Gs",
+                    "matrix": [2, 0],
+                    "x": 0,
+                    "y": 2,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "MI_A",
+                    "matrix": [2, 1],
+                    "x": 1,
+                    "y": 2,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "MI_As",
+                    "matrix": [2, 2],
+                    "x": 2,
+                    "y": 2,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "MI_B",
+                    "matrix": [2, 3],
+                    "x": 3,
+                    "y": 2,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "MO(3)",
+                    "matrix": [3, 0],
+                    "x": 0,
+                    "y": 3,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "TG(1)",
+                    "matrix": [3, 1],
+                    "x": 1,
+                    "y": 3,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "TG(2)",
+                    "matrix": [3, 2],
+                    "x": 2,
+                    "y": 3,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "MI_TOGG",
+                    "matrix": [3, 3],
+                    "x": 3,
+                    "y": 3,
+                    "w": 1,
+                    "h": 1
+                }
+            ]
+        }
```

Comment:
```suggestion
        }
```

Reply:
Ok. I have removed the "LAYOUT_midi" layout from the "keyboard.json" file as requested. Also, this layout was not identical to any of the existing layouts in "keyboard.json" file.